### PR TITLE
Moving emergency phone to a new tagging scheme

### LINKED ISSUE #318: Add rendering for emergency=phone
JOSM validator complains about amenity=emergency_phone being deprecated and auto-corrects it to emergency=phone when using the "Fix" button. However, the latter is not rendered by the carto style, thus all emergency phones disappeared from the map in some areas [1] recently.

I'd suggest to also include emergency=phone with the same render rule as amenity=emergency_phone in the carto style.

[1] Example for a recent mass update turning amenity=emergency_phone into emergency=phone: http://www.openstreetmap.org/changeset/19082658

### PR DESCRIPTION
**Pull request by kocio-pl**
Resolves #318.

Name selection in SQL was removed because we're not displaying names for such POIs.

### CODE REVIEW

File: project.mml [hunk 1]
```diff
@@ -1525,6 +1528,7 @@ Layer:
             OR "natural" IN ('peak', 'volcano', 'saddle', 'spring', 'cave_entrance')
             OR historic IN ('memorial', 'monument', 'archaeological_site', 'wayside_cross')
             OR tags->'memorial' IN ('plaque')
+            OR tags->'emergency' IN ('phone')
```
Comment (pnorman) [inline project.mml]: This [should be `tags @> 'emergency=>phone'`](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#sql-style-guidelines), as well as the repetition in the WHERE clause for the other layer.

Comment (kocio-pl) [reply project.mml]: Hm, but why should we check if a tag is in the hstore? Why do we need this special construction at all?

Comment (pnorman) [reply project.mml]: If we don't include an item in the OR part of the WHERE clause it won't be selected in the layer.

Comment (kocio-pl) [reply project.mml]: The icon on a node was present in my test rendering, so it looks for me that was selected.

Comment (pnorman) [reply project.mml]: The problem is `OR tags->'emergency' IN ('phone')` does not follow our style standards.

Comment (kocio-pl) [reply project.mml]: I see. Does `OR tags->'memorial' IN ('plaque')` have this problem too?

Comment (pnorman) [reply project.mml]: Yes, but you don't need to change it in this PR, it's an existing error.

**Comment by kocio-pl** [comment]
I have changed SQL formatting as requested. 

I have also removed support for rendering such polygons, because there are only 34 of them, Wiki definition does not allow areas and they all look like mistagging (property of building or part of a road).

**Pull request merged**